### ISSUE TITLE
Is there a reason to use train/validation/test split of the data?

### ISSUE THREAD

**Issue by Aoobi** [ultralytics/yolov5#3233]
## ❔Question
Hi, 
I am just wondering if images from validation set are used for training in any way? I am asking because I don't know if I can test the model on validation set, or do I need a separate "test" set?

Thanks!

**Comment by glenn-jocher** [comment]
@Aoobi validation set is not used during training other than to generate metrics for logging and for saving best.pt. 

You don't need a test set, it's typically there for competitions where the test labels are not provided and only known privately to prevent overfitting the test set.

**Comment by Aoobi** [comment]
Thank you :)
